resharper/src/resharper-unity/Feature/Services/LiveTemplates/UnityProjectScopeCategoryUIProvider.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System.Collections.Generic;
+using JetBrains.ReSharper.Feature.Services.LiveTemplates.Scope;
+using JetBrains.ReSharper.Feature.Services.LiveTemplates.Templates;
+using JetBrains.ReSharper.Plugins.Unity.Feature.Services.LiveTemplates.Scope;
+using JetBrains.ReSharper.Plugins.Unity.Resources;
+using JetBrains.ReSharper.Psi.CSharp.Resources;
+
+namespace JetBrains.ReSharper.Plugins.Unity.Feature.Services.LiveTemplates
+{
+    // Defines a category for the UI, and the scope points that it includes
+    [ScopeCategoryUIProvider(Priority = -200, ScopeFilter = ScopeFilter.Project)]
```

Comment:
What does `-200` mean here? Do we have some constant in SDK to use instead of that? :)

Reply:
Sadly, no. It just needs to be less than existing categories, so it appears last in the list in the ReSharper Template Editor. Rider seems to group by quick list, possibly alphabetically, but it ignores this. I'll add a comment.